Jenkinsfile

```diff
@@ -58,19 +67,25 @@ pipeline {
         }
         steps {
           container('maven') {
-            dir ('./charts/activiti-cloud-full-example') {
+            dir ("./charts/$APP_NAME") {
               sh 'jx step changelog --version v\$(cat ../../VERSION)'
               // promote through all 'Auto' promotion Environments
               sh 'jx promote -b --all-auto --helm-repo-url=$GITHUB_HELM_REPO_URL --timeout 1h --version \$(cat ../../VERSION) --no-wait'
-              sh 'cd ../.. && updatebot push-version --kind helm activiti-cloud-full-example \$(cat VERSION)'
+             // sh 'cd ../.. && updatebot push-version --kind helm $APP_NAME \$(cat VERSION)'
             }
           }
         }
       }
     }
-    post {
+   post {
         always {
-            cleanWs()
+          container('maven') {
+            sh "kubectl delete namespace $PREVIEW_NAMESPACE" 
+            dir("./charts/$APP_NAME") {
```

Comment:
@almerico I think it will make more sense to delete Helm preview release before deleting preview namespace.

Reply:
@igdianov forgot to remove it was tests in a second will provide final version